### PR TITLE
Update DG

### PR DESCRIPTION
**Pull request by jktang14**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -467,6 +467,7 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
 8. **Reject email inputs without a period between alphanumeric characters for the domain portion:** Currently, an input like `e/john@example` is accepted, however, this should not be the case as a valid domain format would require at least 1 period for the top level domain (ie.john@example.com). We plan to adjust the regex for the domain portion of the email from `"(" + DOMAIN_PART_REGEX + "\\.)*"` to `"(" + DOMAIN_PART_REGEX + "\\.)+"` to reject such an input.<br>
+9. **Make `INDEX` 0 and negative `INDEX` error messages more specific:** Currently, when 0 or negative numbers are passed for the `INDEX` parameter, the error message thrown is `Invalid command format!`. In the future, we will change this error message to `The person index provided is invalid`.
```
Comment (JunLongling) [inline docs/DeveloperGuide.md:470]:
```suggestion
9. **Make `INDEX` 0 and negative `INDEX` error messages more specific:** Currently, when 0 or negative numbers are passed for the `INDEX` parameter, the error message thrown is `Invalid command format!`. In the future, we will change this error message to `The person INDEX provided is invalid`.
```

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -291,7 +291,7 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | `* *`    | software developer | blacklist a client                           | avoid working with certain clients in future                       |
 | `* *`    | software developer | view clients on the blacklist                | view all the clients I intend to avoid                             |
 | `* *`    | software developer | whitelist a blacklisted client               | consider working with said client again                            |
-| `* *`    | software developer | view clients on the whitelist                | view all clients that are not on the blacklist                     |
+| `* *`    | software developer | view clients on the whitelist                | view all clients that i have no issues working with                |
```
Comment (rahula1008) [inline docs/DeveloperGuide.md:294]: NIT: view all clients that *I* have no issues working with

File: docs/AboutUs.md [hunk 3]
```diff
@@ -46,7 +46,7 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 [[portfolio](https://www.linkedin.com/in/tang-jun-kiat-0540582b6/)]
 
 * Role: Developer
-* Responsibilities: Dev Ops + Threading
+* Responsibilities: Deliverables and deadlines
 
 ### Prakamya Singh
```
Comment (Dinoman44) [inline docs/AboutUs.md:51]: Make my responsibilities as `Commands, Code Quality` and role as `QA (Quality Assurance)`

File: docs/AboutUs.md [hunk 4]
```diff
@@ -27,7 +27,7 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 
 * Role: Developer
-* Responsibilities: Data
+* Responsibilities: Code quality
```
Comment (Dinoman44) [inline docs/AboutUs.md:30]:
```suggestion
* Responsibilities: Commands, UX
```

File: docs/AboutUs.md [hunk 5]
```diff
@@ -27,7 +27,7 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
```
Comment (Dinoman44) [inline docs/AboutUs.md:27]: Make Tushar's responsibilities `GUI, Commands`

**Review by Dinoman44 — COMMENTED** [review]
Few nits but otherwise LGTM

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (jktang14) [reply docs/DeveloperGuide.md:470]: Our current index message is not caps so i standardised

**Review by Dinoman44 — APPROVED** [review]

**Pull request merged**